application/controllers/collection_instrument.py

```diff
@@ -77,7 +76,10 @@ def upload_instrument(self, exercise_id, file, ru_ref=None, classifiers=None, se
         log.info('Upload exercise', exercise_id=exercise_id)
 
         validate_uuid(exercise_id)
-        instrument = self._create_instrument(file)
+        instrument = InstrumentModel(ci_type='SEFT')
```

Comment:
Should we be able to specify 'type' here or is that something that is being picked up by Emilio's PR?